openslides_backend/action/actions/meeting/import_.py

```diff
@@ -575,7 +575,7 @@ def create_events(
         )
 
         # add meetings to organization if set in meeting
-        adder: Dict[str, List[Union[int, str]]] = {}
+        adder: ListFieldsValue = {}
```

Comment:
Please revert this file. A type ListFieldsValue for a dict is the opposite of readable

Reply:
I still think the best is to revert it.
I asked myself, why it is a union? One would expect the meeting_id to be always int.

Reply:
`Dict` is [invariant](https://mypy.readthedocs.io/en/stable/common_issues.html#variance) and the `list_fields` can potentially be used with integers and strings, therefore, the type used here must be a union as well. But this is actually an argument *for* a type alias: You see a union of strings and integers and are confused at first. If we just use some type alias, this confusion can be avoided. Additionally, this rather verbose type is used in 3 separate places, so it also goes against the DRY principle to repeat the type. As I said, I'm open for another name suggestion if `ListFieldsValue` does not suit you.

Reply:
renamed to `ListFieldsDict`